### ISSUE TITLE
Add RF dropouts

### ISSUE THREAD

**Issue by Break8731** [valadaptive/ntsc-rs#68]
There are still many VHS artifacts missing so this could be a start, here's also some examples 
![image-13](https://github.com/valadaptive/ntsc-rs/assets/161225418/9e7a337a-9b07-4a0c-a626-6687aa8fff21)
![image-16](https://github.com/valadaptive/ntsc-rs/assets/161225418/e9c8aeb1-fa4e-4987-a2ef-906cb81f64ef)

**Comment by nlolnlolnlolnlo** [comment]
this looks like the head switching artifact moving upwards

**Comment by Break8731** [comment]
Nope, It just appears and disappears it doesn't seem to move up and down, when I come back home I'll also send a video here

**Comment by Break8731** [comment]
> this looks like the head switching artifact moving upwards

Nope, It just appears and disappears it doesn't seem to move up and down, when I come back home I'll also send a video here

**Comment by valadaptive** [comment]
I've been meaning to implement this for a while, since I've noticed it in a lot of VHS recordings, but I still don't know how it works fully. It doesn't seem like the VCR necessarily repeats the previous line always, but it doesn't look exactly like head switching artifacts either.

**Comment by Break8731** [comment]
> A VCR dropout occurs when the tape emulsion fails. It can be on one line and can last for a few nanoseconds, Most notably found in deinterlaced footage. It could be implemented under the "VHS" tab. This could be made with the color loss logic.

I found an article that explains more in detail the causes of white noise and other type of dropouts and how to prevent them.



https://cool.culturalheritage.org/videopreservation/vid_guide/6/6.html

**Comment by UltraBlackLinux** [comment]
I'd assume it's this? https://www.avartifactatlas.com/artifacts/scratches_and_tape_wear.html
The second example video looks pretty similar to what you're seeing there
